styles/themes/theme.less

```diff
@@ -101,26 +157,11 @@ h1, h2, h3, h4, h5, h6 {
     background-color: @sidebar-background;
 }
 
-/* Attention-getting structures */
-.error, .test_warning {
-    color: red;
-}
-
-.warning {
-    color: blue;
-}
-
-.highlight {
-    background-color: yellow;
-    color: black;
-}
-
-.button {
-    .sans-serif;
-}
 
-kbd {
-    color: #cc0000;
+/* ------------------------------------------------------------------------ */
+/* News items */
+div.newsitem {
+    box-shadow: -5px -5px 5px lightgrey; //srj: moved from layout
```

Comment:
Remove your note to self.

Reply:
Done. (Darn, I thought I'd gotten them all!)